testdata/serviceconfig.yml

```diff
@@ -21,7 +21,7 @@ rules:
           method: api.method | "unknown"
           response_code: response.http.code | "unknown"
       - descriptor_name:  request_latency
-        value: response.latency | "unknown"
+        value: response.latency | 0
```

Comment:
Prometheus handles "NaN" correctly (although, the counter permanently becomes NaN afterwards).  0 is fine for now (it will make our averages weird if we aren't careful though).

In the end, we need a histogram/distribution type for latency -- one that can include counts distinct from relying on request_count metrics for calculations, etc.